src/command/init.rs

```diff
@@ -188,9 +188,25 @@ impl Init {
         step: &Step,
         log: &Logger,
     ) -> result::Result<(), Error> {
-        info!(&log, "Installing wasm-bindgen-cli...");
-        bindgen::cargo_install_wasm_bindgen(step)?;
-        info!(&log, "Installing wasm-bindgen-cli was successful.");
+        let bindgen_version = manifest::get_wasm_bindgen_version(&self.crate_path)?
+            .expect("Unexpected error while selecting wasm-bindgen version");
+
+        let mut install_needed = true;
+        info!(&log, "Searching for locally installed WASM-bindgen...");
+        if bindgen::wasm_bindgen_locally_installed(&self.crate_path) {
+            info!(&log, "Checking WASM-bindgen version...");
+            if let Some(version) = manifest::get_wasm_bindgen_version(&self.crate_path)? {
+                if bindgen::wasm_bindgen_version_check(&version)? {
+                    install_needed = false;
+                }
+            }
+        }
+
+        if install_needed {
+            info!(&log, "Installing wasm-bindgen-cli...");
+            bindgen::cargo_install_wasm_bindgen(&self.crate_path, &bindgen_version, step)?;
+            info!(&log, "Installing wasm-bindgen-cli was successful.");
+        }
```

Comment:
The control flow here is the main thing I'd like to clean up, not especially happy with these nested `if` statements.